uwazi/entity-page/src/lib/DownloadDialog.svelte

```diff
@@ -0,0 +1,124 @@
+<!-- Download attestation to personal hyperbee modal -->
```

Comment:
oh, interesting! I see, downloading to your own hyperbee, rather than to the UWAZI db. See my comment on the "clone/copy" icon to see how I had been thinking about this. Writing to one's own Hyperbee makes a lot of sense.

Reply:
Oh I see I misread your comment all the way back in May, you were talking about adding Uwazi metadata. I can change it to that, but I think this option has value as well. Let me know what you think.

Reply:
I do think we need to integrate with the UWAZI db at some point, but that requires 1) knowing more about how users are making use of UWAZI, and 2) a deeper UWAZI integration that we might not have control over. 

I'm fine sticking with copying to our hyperbee instance, but in that case, it does raise questions about whether the copier is re-signing and re-timestamping and making the attestation their own, or whether the copier is just transferring data. Does the server have a way to write to the hyperbee without signing/timestamping? That sounds like a new feature to me. I am ok dropping the copy/download feature entirely for now.

Reply:
As currently implemented it is re-signing and re-timestamping. That method makes sense to me, because it allows you to throw your own weight behind something someone else said. Maybe the BBC is copying something their photographer posted, but they are re-signing it with their key, which has much more credibility.

Anyway if you think we should drop the feature I can do it. It might make sense to check with Ben first though, since I have discussed it with him previously.

Reply:
By BBC "re-signing" do you mean that BBC is signing over the attestation already signed by their photographer? If so, that sounds like a good use case to me. Since attestations have their own CID, we can do this by using the attestation CID as the key prefix. 

I don't think re-signing and re-timestamping the raw text is as useful. It's taking it out of its original context which means it loses important context.

Reply:
I didn't mean that, but I see how that's useful. Probably the right way to go. For now I will remove the feature as discussed and further discussion can continue in #38.